subprojects/plugin-development/src/integTest/groovy/org/gradle/plugin/devel/impldeps/GradleImplDepsLoggingIntegrationTest.groovy

```diff
@@ -47,12 +47,6 @@ class GradleImplDepsLoggingIntegrationTest extends BaseGradleImplDepsIntegration
 
         then:
         def gradleVersion = GradleVersion.current().version
-        output.contains("Generating JAR file 'gradle-api-${gradleVersion}.jar'")
-        output.contains("Generating JAR file 'gradle-test-kit-${gradleVersion}.jar'")
```

Comment:
Users actually never see progress for testkit, because it is just one jar, so shading it completes almost immediately.